daemon/execdriver/lxc/driver.go

```diff
@@ -214,10 +256,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 	c.ProcessConfig.Args = append([]string{name}, arg...)
 
 	if err := createDeviceNodes(c.Rootfs, c.AutoCreatedDevices); err != nil {
+		killNetNsProc(proc)
```

Comment:
It's cool that we now wait, but I find it unfortunate that it's called in each code path rather than `defer`ed: is there a reason not to?